.github/workflows/tutorials.yml

```
@@ -47,10 +47,48 @@ jobs:
       env:
         SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
 
-  sdk-estimate-costs:
+  send-tx-from-eth:
     needs: cross-dom-bridge-eth
     runs-on: ubuntu-latest
 
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)